plugins/inputs/filecount/filecount.go

```diff
@@ -203,6 +208,13 @@ func (fc *FileCount) count(acc telegraf.Accumulator, basedir string, glob globpa
 		Callback:             walkFn,
 		PostChildrenCallback: postChildrenFn,
 		Unsorted:             true,
+		ErrorCallback: func(osPathname string, err error) godirwalk.ErrorAction {
+			if os.IsPermission(errors.Cause(err)) {
+				log.Println("I! [inputs.filecount]", err)
```

Comment:
Use `D!`, since I think most will just want to ignore this error.  Can you paste an example of how the error looks here, we might need some leading context.

Reply:
Meant to do that,
```
2019-02-25T23:33:30Z D! [inputs.filecount] cannot ReadDirents: cannot Open: open /tmp/foo/bar: permission denied
```